Sofa/framework/Core/src/sofa/core/objectmodel/Base.h

```diff
@@ -369,6 +369,8 @@ class SOFA_CORE_API Base
     mutable std::deque<sofa::helper::logging::Message> m_messageslog ;
 
 public:
+    mutable Data<int> d_messageLogCount;
```

Comment:
I don't see where this Data is initialized (usually in the constructor with `initData`). Isn't it required?

Reply:
The initData is an helper class to gives the data a public name, an owner and register it to the list of public data of a Base. If not used the strings are "", and owner = nullptr. 
Without saying it is a good idea I don't remember having see that clearly stated that a data can/cannot be used that way, but currently it is not enforced at the API side.

Reply:
ok ok I see. I just wanted to be sure.